Gemfile.lock

```diff
@@ -94,7 +94,7 @@ GEM
     listen (3.8.0)
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
-    lograge (0.12.0)
+    lograge (0.14.0)
```

Comment:
grit: are all the dependencies updated/patched to address any security concerns etc or are they to get the devenv up and running only atm?  (will we patch/address dependabot issues in another ticket?).

Reply:
The latter. Updating dependencies is separate in the backlog